src/A1comms/GaeSupportLaravel/Integration/TaskQueue/Client.php

```diff
@@ -55,6 +55,8 @@ private function fetchLocation() {
             case "eu5":
             case "eu6":
                 return "europe-west1";
+            case "us6":
+                return "us-central1";

```

Comment:
Do you mind removing line 59 to match the select/case style from the European zones please?